codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionEncoder.java

```diff
@@ -202,7 +200,7 @@ public void operationComplete(ChannelFuture future) throws Exception {
                 // Pass headers to the flow-controller so it can maintain their sequence relative to DATA frames.
                 flowController.addFlowControlled(stream,
                         new FlowControlledHeaders(stream, headers, streamDependency, weight, exclusive, padding,
-                                                  endOfStream, promise));
+                                                  true, promise));
```

Comment:
yah it will always be false here. this is an `else` statement where the `if` includes a `! endOfStream` clause. Do you see an issue?